test/CMakeLists.txt

```diff
@@ -160,7 +160,7 @@ function(add_test_command NAME EXE)
         endif()
     endif()
     
-    if(WORKAROUND_ISSUE_898 AND MIOPEN_BACKEND_USE_COMGR)
+    if(WORKAROUND_ISSUE_898 AND MIOPEN_USE_COMGR)
         set_property(TEST ${TEST_NAME} PROPERTY ENVIRONMENT MIOPEN_DEBUG_COMGR_HIP_PCH_ENFORCE=0)
```

Comment:
@shurale-nkn It should be `set_property(TEST ${NAME}...` isn't it? I am wondering how this works...

Reply:
@atamazov 
Yes, in this function better to use ${NAME}.

But it works because function can access to scope defined before the moment then function was called.
So add_test_command was called from add_test_executable where TEST_NAME is defined as argument, and because add_test_executable is not finished the scope is not deleted. And the TEST_NAME variable accessible from add_test_command.

Reply:
Oh great CMake syntax!